config/routes.rb

```diff
@@ -10,4 +10,10 @@
   namespace :admin do
     resources :users
   end
+
+  namespace :api do
+    namespace :v1 do
+      resources :tasks, except: [:new, :edit]
```

Comment:
we prefer to use allowlist instead of blocklist for routing. the allowlist is easy to understand and it does not require thinking of what will work after the blocklist